### PR TITLE
Provide a Dockerfile.

### PR DESCRIPTION
**Pull request by janLo**
This adds a Dockerfile for easy installation as docker image.
Usage documentation is included in the README.

The branch is based on the webdav branch of PR #19 .

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,6 +77,26 @@ Edit this file and change the interface/port as needed.
...
 
+  - Docker
+
+Build the image in the applications root dir:
+
+```
+docker build -t <image_name> .
+```
+
+The image can be confiured via environment variables. The available variables are:
```
Comment (EpocDotFr) [inline README.md]: Typo: confiured

**Comment by EpocDotFr** [comment]
I'm not using Docker at all nor have the need to add Docker support in this project nor have the desire to learn it, **however** I can merge your PR if you mention somewhere in the readme that Docker support is maintained by you, and issues related to Docker must be reported to you.

**Comment by EpocDotFr** [comment]
Also don't forget to credit you for Docker support in the readme :)

**Comment by janLo** [comment]
I mentioned myself in the README along with the docker support and fixed the typo.

**Comment by EpocDotFr** [comment]
Can you merge master so I can merge this PR?

**Comment by janLo** [comment]
I rebased it on top if the latest master and changed the webdavclient3 to the released pip version as the necessary fix should now be included.

**Comment by EpocDotFr** [comment]
Thanks! Merged.

**Pull request merged**